README.md

```diff
@@ -68,23 +68,23 @@ roleRef:
+| spec.param.trafficGeneratorImage            | Traffic generator's container image                                                                               | False        | Defaults to the U/S image https://quay.io/repository/kiagnose/kubevirt-dpdk-checkup-traffic-gen:latest |
+| spec.param.trafficGeneratorNodeSelector     | Node Name on which the traffic generator Pod will be scheduled to                                                 | False        | Assumed to be configured to Nodes that allow DPDK traffic                                              |
+| spec.param.trafficGeneratorPacketsPerSecond | Amount of packets per second in Millions                                                                          | False        | Defaults to 14                                                                                         |
+| spec.param.trafficGeneratorEastMacAddress   | MAC address of the NIC connected to the traffic generator pod/VM                                                  | False        | Defaults to 50:00:00:00:00:01                                                                          |
+| spec.param.trafficGeneratorWestMacAddress   | MAC address of the NIC connected to the traffic generator pod/VM                                                  | False        | Defaults to 50:00:00:00:00:02                                                                          |
+| spec.param.vmContainerDiskImage             | Container disk image for the VM                                                                                   | False        | Defaults to https://quay.io/repository/kiagnose/kubevirt-dpdk-checkup-vm:latest                        |
+| spec.param.DPDKLabelSelector                | Node Label of on which the VM shall run                                                                           | False        | Assumed to be configured to Nodes that allow DPDK traffic                                              |
+| spec.param.DPDKEastMacAddress               | MAC address of the NIC connected to the DPDK VM                                                                   | False        | Defaults to 60:00:00:00:00:01                                                                          |
+| spec.param.DPDKWestMacAddress               | MAC address of the NIC connected to the DPDK VM                                                                   | False        | Defaults to 60:00:00:00:00:02                                                                          |
+| spec.param.testDuration                     | How much time will the traffic generator will run. format: <amount>[/k/m] no-suffix:seconds m:minutes h-hours     | False        | Defaults to 5m                                                                                         |
```

Comment:
What does `k` stands for?
I suggest to use Go's time.Duration [conventions](https://pkg.go.dev/maze.io/x/duration#ParseDuration), and internally convert it to whatever trex wants.

Reply:
I agree. We actually already do this and I forgot. 

Closing this PR